### PR TITLE
Add campaign tracking data

### PR DESCRIPTION
**Pull request by mikethetike**
### Closes Issues:
Connected to big-neon/bigneon#49

### Description:
Adds fields for tracking campaign data. The frontend must extract URL parameters and add `tracking_data` to the cart methods  

### Migrations
 * 20190418121122_add_tracking_data_to_order

### Environment Variables
 * No change

### CODE REVIEW

File: db/migrations/20190418121122_add_tracking_data_to_order/down.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+ALTER TABLE orders
+    DROP tracking_data;
+ALTER TABLE orders
```
Comment (Krakaw) [inline db/migrations/20190418121122_add_tracking_data_to_order/down.sql]: Purely a style thing, can this not be done in a single `ALTER TABLE orders`?

File: db/migrations/20190418121122_add_tracking_data_to_order/up.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+ALTER TABLE orders
```
Comment (Krakaw) [inline db/migrations/20190418121122_add_tracking_data_to_order/up.sql]: Same as above, purely a style thing, can this not be done in a single `ALTER TABLE orders`?

File: db/migrations/20190418121122_add_tracking_data_to_order/down.sql (hunk 1 above)
Comment (mikethetike) [reply db/migrations/20190418121122_add_tracking_data_to_order/down.sql]: You can do that?

Comment (Krakaw) [reply db/migrations/20190418121122_add_tracking_data_to_order/down.sql]: `ALTER TABLE table DROP COLUMN col1, DROP COLUMN col2;`

**Pull request merged**